contracts/deployment/SystemV2.sol

```diff
@@ -240,7 +241,7 @@ contract SystemV2 {
         daiConstantOracle = new ConstantPriceOracle(address(core), 1e18);
         usdcConstantOracle = new ConstantPriceOracle(
             address(core),
-            1e18 * 10 ** uint256(uint8(USDC_DECIMALS_NORMALIZER))
+            1e18 * 10 ** uint256(uint8(USDC_DECIMALS_NORMALIZER)) /// todo this doesn't look correct, verify
```

Comment:
should be 1e30, which is correct ? 1e6 USDC wei is woth 1$, and we express values with 18 decimals, so 1e6 USDC * 1e30 oracle / 1e18 = 1e18 = 1$

Reply:
you can tell it's correct in the PCVOracle integration tests, 100 DAI (100e18 DAI base units) is worth 100$, and so is 100 USDC (100e6 USDC base units)